css/main.css

```
@@ -6,5 +6,20 @@
 body {
   font-family: 'Poppins', sans-serif;
   color: #3b3d40;
-  font-size: 14px;
+  font-size: 14px; 
+} 
+
+header{ 
```

Mudar nome da classe para:
```suggestion
.organizacao-cabecalho{ 
```